app/controllers/web/dashboard/users_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Web
+  module Dashboard
+    # Assigns the role to the user
+    class UsersController < BaseController
+      before_action :user_find, except: :index
+      before_action :authorize_role
+      rescue_from Pundit::NotAuthorizedError, with: :redirect_to_dashboard_users
```

Comment:
For simplicity let's move that to `Dashboard::BaseController` and always redirect to `dashboard_root_url`

Reply:
fix